File: distributed-workflow-engine/src/app/model/workflow.ts
import { Task } from "./task";

export class WorkFlow{
    workFlowName:String;
    owner: String;
    canViewUser: Array<String>;
    canEditUser: Array<String>;
    workFlowStatus: String;
    tasks: Map<String,Task>;

    constructor(workFlowName, owner, canViewUser, canEditUser, workFlowStatus, tasks){
        this.workFlowName = workFlowName;
        this.owner= owner;
        this.canViewUser = canViewUser;
        this.canEditUser = canEditUser;
        this.workFlowStatus = workFlowStatus;
        this.tasks = tasks;
    }
}

File: Engine_dw-state-initializer/src/main/java/com/distributedworkflowengine/stateinit/domain/Workflow.java
package com.distributedworkflowengine.stateinit.domain;

import java.util.List;

//import javax.persistence.Column;

public class Workflow {
	private String workFlowName;
	
	private List<Task> tasks;

	public String getWorkFlowName() {
		return workFlowName;
	}

	public void setWorkFlowName(String workFlowName) {
		this.workFlowName = workFlowName;
	}

	public List<Task> getTasks() {
		return tasks;
	}

	public void setTasks(List<Task> tasks) {
		this.tasks = tasks;
	}
}
